yt_dlp/extractor/tiktok.py

```diff
@@ -467,6 +460,7 @@ class TikTokIE(TikTokBaseIE):
             'uploader_id': '7036055384943690754',
             'uploader_url': 'https://www.tiktok.com/@MS4wLjABAAAATh8Vewkn0LYM7Fo03iec3qKdeCUOcBIouRk1mkiag6h3o_pQu_dUXvZ2EZlGST7_',
             'track': 'Promoted Music',
+            'artist': '',
```

Comment:
This appears to be a bug in the code. Adding a `or None` where this extraction is happening should fix it

Reply:
fixed